### DISCUSSION TITLE
Governance Weekly Meeting 27th May 2021

### DISCUSSION THREAD

**Discussion by tttroyyy** [NFT-DAO/Governance-HOLON#108] [Meetings]
**NFT-DAO GOVERNANCE HOLON MEETING (NOTES)**
THURSDAY MAY 27 2021 7.00PM UTC 

Attendees: Hud, Sev, Troy, Rareș & Anamaria (New Mindflow), John 

MEETING OUTLINE

- Decision-Making Protocols
- Contributions (Sourcecred update)
- Legal issues (operating agreements and tokenomics)
- Open Discussion

DECISION-MAKING PROTOCOLS

- TROY: Introducing a decision-making problem that needs to be resolved as we seek a process that considers how we reach consensus and between how many participants? We have experimented with a basic Sociocracy approach (which was time consuming), polls and informal decision-making amongst team leads and holon participants, which has affected progress;
- NEWMINDFLOW: The Contributions/OccamFi poll had two topics: Limit polls to one topic at a time. A problem is that big decisions are being made unilaterally without consultation;
- HUD: Problems around coherence when different ideas are posed; Lack of clarity around whether NFT-DAO or the Boxcar has priority; I like the idea of polls; People who vote need a way to be well informed, need a way to agree to terms of the poll, like a checkbox confirming users know the poll parameters;
- TROY: Confirmed that the Boxcar is the primary product and the DAO was a suggested means of achieving the product, which is more complex and will take more time to implement;
- NEWMINDFLOW: Making decisions as a group is where we are now; Concrete decisions have been made in a unilateral process which has impacted on the DAO;
- TROY: Reiterated that the DAO has employed an experimental and iterative approach from the outset which has been a process of trial and error. Also that our decision-making process is still in the “Forming” stage, and ideological conflicts (differences of opinion about how things should be done and by whom) are part of the “Storming” stage. We want to formalise our processes into the “Norming” and “Performing” stages (to paraphrase Rich);
- SEV: a suggestion to continue with polls as a decision-making tool; Prefers to seek majority voting not a quorum;
- TROY: Rich forwarded a suggestion that 16% the Ticket Sales be redistributed back to the 16 core contributors (identified by the core holon prior to members exiting the DAO). This would amount to 1% distributed to core members (or roughly 5000 ADA if all tickets are sold);
- NEWMINDFLOW: Raised a question about how those core contributors were selected, and also what will happen to the remaining profits from the sale;
- TROY: described that the 16 contributors were identified from an informal discussion between core holon members. This came about with a desire to reward contributors who were continually and consistently contributing to projects leading to project outcomes. Troy identified Sev, Hud, Stephen and Dan as core contributors in the gov. holon at the time; It was shared that contributors could change over time as some chose to leave or join, and a staged approach to governance and engagement would need to be considered;
- TROY: Shared the draft token distribution model as part of the business model draft as a proposed way to split profits into the NFT-DAO treasury.

CONTRIBUTIONS (SOURCECRED UPDATE)

- HUD & SEV: Outlined how Sourcecred can be applied to Discord and Github; 
- NEWMINDFLOW: Looks like sourcecred will be limited in what it can capture and doesn’t capture extra work, contributions that happen on or off the server; Will be more to gamify the system to increase community engagement;
- TROY: in the context of the EZ-Honor prop, sourcecred would act as a measure for “kudos” within the community. This is a lower tier form of community engagement. Governance system requires a voting mechanism, and an operational agreement will also help establish community trust; 
- QUESTION: Will we continue exploring sourcecred as a tool to support “kudos” and community engagement? (General informal consensus Sev, Hud, Troy: Yes)
- TROY: there is a Sample DAO Charter available here https://medium.com/token-suisse/sample-dao-charter-c59f5396ceea 

LEGAL ISSUES

- TROY: Discussed Operational Agreement pathway from yesterday’s General meeting: https://github.com/NFT-DAO/Governance-HOLON/discussions/106#discussion-3383929 
- TROY: A legal “holon” meeting happened this morning with Kyle (Phillip Robert’s lawyer) who has joined the DAO to support legal discussions;
- NEWMINDFLOW: It was an introductory meeting and the key takeaway was that instead of a interim operational agreement, a proper agreement should be drafted once the Wyoming State legislature passes the DAO law in July 1;
- NOTE: Rich is attending a Wyo State hearing today. His testimony is here: https://youtu.be/9IWbi6__PbU 

OPEN DISCUSSION

- NEWMINDFLOW: For a DAO we need aspects of governance embedded in smart contracts: “Dealing with Articles of Organization” https://wyoleg.gov/2021/Introduced/SF0038.pdf 
- NEWMINDFLOW: Total transparency needed
- TROY: We also need to be clear about the parameters of transparency within the organisation, since transparency is not total (ie, direct messaging, proprietary/commercial products, marketing planning, trade secrets, etc). Having clarity with members who have different expectations of how transparency should look can help us to avoid future problems;
- SEV: We need to get things done. A code of conduct can stipulate transparency requirements;
- TROY: Since inception, we have been moving from an open start-up approach (with four core founders) with a vision of moving toward a shared governance model. That means that no one had entered into any formal agreements and contributed through goodwill, and that the desire is to build and share governance and rewards in a staged approach: 1) Plus Ones (core contributors), 2) Plus Two’s (the next generation of core contributors), 3) Plus Three’s (and so forth);
- NEWMINDFLOW: Regarding a poll, Stephen’s self-assessment proposal is the best approach: https://github.com/NFT-DAO/Governance-HOLON/discussions/96
- TROY: Challenging the notion of “best approach” as this may be subjective. It is proposed that a poll for core contributors can include options for determining how to distribute contributions including ones proposed by Stephen, Rich’s suggestion, and any other proposed solutions. This then begs the question of how will proposals look?
- NEWMINDFLOW: Discussed the dev team leaving and how this was prompted by how the way they were communicated to made them feel expendable;
- JOHN: Is there a way to mediate or resolve the dev relationship;
- TROY: Clarified that three devs had left after a contribution offer was proposed; also that tension had mounted when a one-week deadline approached and there was little prior communications between the dev team and other holons. Unfortunately, fallout resulted on both sides when code wasn’t being released with an impending deadline, and unfortunate words were said. Also since no-one in this meeting was present in the discussion, we are only speculating. The takeaway: communicate earlier, formalise terms and show gratitude to one another.

QUESTIONS ARISING:

- What constitutes a Major or Minor decision? (ie, number of participants involved, scope the decision will affect, etc)
- Who will vote/make the decision? (ie, core team, core contributors, Discord server, holons, meeting members, etc)
- For majority voting, what will be done in the case of split decisions?
- Who are the core contributors now? How will this be determined over time?
- What will happen to the remaining profits from the Ticket sales?
- How will proposals be generated and forwarded to key voters?
- What decision needs to be made regarding contributions to key contributors and by whom?
- How will we finalise the participant range and issue importance for polls?

ACTION ITEMS (LIST)
WHAT DO WE NEED? (Tasks, Assignments & Timeframes)

ACTIONS							                                 DRIVERS			TIMEFRAME

- Draft decision-making pathway 	 			                  @tttroyyy / All 	ASAP
- Brainstorm/draft a decision pathway (Google.doc)	          All				ASAP
- Brainstorm/draft a contribution proposal (Google.doc)	  All				ASAP		
- Research needed around decision-making processes 
for collaborative groups		                                                  @hududed 
- Research on DAO-maker organisations (Aragon)		  @severoni1573 
- Continue Sourcecred exploration 			                  @hududed / All

MEETING ENDED: 9.50PM UTC

ADDITIONS:

- NEWMINDFLOW: 'Making decisions as a group is what we should strive for' (28-5-21 UTC)

**Comment by newmindflow** [comment]
Hi Troy, could you please amend the notes to include our actual artist name/alias/commercial name: New Mindflow instead of NEWMIND.
If nothing else to maintain consistency between records.
Or if you want to use our personal names as you did in the attendees list you could go with Rareș and Anamaria (New Mindflow)

A correction to the notes. Maybe we haven’t expressed it clearly enough and it’s been misinterpreted, but Making decisions as a group is where we are now was kind of exactly the opposite of what we are trying to express so probably the notes should mention Making decisions as a group is what we should strive for.

Another correction I would like to point out is that I didn't suggest Stephen’s self-assessment survey as an example of how a poll/survey should be conducted but rather as the best method that we currently have to assess members' contributions.
https://github.com/NFT-DAO/Governance-HOLON/discussions/96

**Reply by tttroyyy** [reply]
"Hi Troy, could you please amend the notes to include our actual artist name/alias/commercial name: New Mindflow instead of NEWMIND.
If nothing else to maintain consistency between records.
Or if you want to use our personal names as you did in the attendees list you could go with Rareș and Anamaria (New Mindflow)"

DONE

"A correction to the notes. Maybe we haven’t expressed it clearly enough and it’s been misinterpreted, but Making decisions as a group is where we are now was kind of exactly the opposite of what we are trying to express so probably the notes should mention Making decisions as a group is what we should strive for."

ADDITION ADDED AT THE VERY END.

"Another correction I would like to point out is that I didn't suggest Stephen’s self-assessment survey as an example of how a poll/survey should be conducted but rather as the best method that we currently have to assess members' contributions.#96"

SELF-ASSESSMENT AS AN EXAMPLE IS A POINT I WAS MAKING: "TROY: Challenging the notion of “best approach” as this may be subjective. It is proposed that a poll for core contributors can include options for determining how to distribute contributions including ones proposed by Stephen, Rich’s suggestion, and any other proposed solutions. This then begs the question of how will proposals look?"

**Comment by newmindflow** [comment]
While it is true that we weren't present in the meeting that ultimately led to the departure of several members of the development team, to qualify what we expressed in the governance meeting as speculative would be inaccurate as they are the result of multiple interviews and long discussions with almost all parties that _were_ directly involved in that meeting.

**Reply by tttroyyy** [reply]
Hey guys. I think the point was more about how none of us (not just you) were in that particular meeting so its hard to properly gauge the intent of others who aren't there to speak for themselves. Our second hand interpretation of the events is what I was considering speculative.
